teamB/jeongyunah/controllers/postController.js

```diff
@@ -0,0 +1,93 @@
+const postService = require('../services/postService')
+
+const postUp = async (req, res) => {
+     try{
+        const { title, content, user_id} = req.body;
+
+        if (!title || !content || !user_id) {
+            return res.status(400).json({message : 'KEY_ERROR'});
+        }
+v
+        await postService.postUp( title, content, user_id);
+        return res.status(201).json({
+            message : 'POSTUP_SUCCESS'
+        });
+       }catch (err) {
+            console.log(err);
+            return res.status(err.statusCode || 500).json({message : err.message});
+        }
+    };
+
+    const postLookup = async (req, res) => {
```

Comment:
해당 함수 이름도 다음과 같이 고치면 좋을 것 같아요.
```suggestion
    const getPostList = async (req, res) => {
```